assignment5/src/components/Body.js

```diff
@@ -0,0 +1,36 @@
+import DeveloperCard from "./DeveloperCard";
+
+function Body(props) {
+  const { userData } = props;
+  return (
+    <section>
+      <div className="developers">
+        {userData.map((developer) => (
+          <DeveloperCard
```

Comment:
hey vicky, here we can pass props of developer only once and can do obj destructuring in developer card comp. 
like this.    <DeveloperCard key={developer.id} details={developer} />
so that we can do destructuring for all those keys inside it in component, instead of writing so many times as developer.xxxx.

Reply:
will keep in mind in upcoming assignments.